hvplot/__init__.py

```diff
@@ -1,3 +1,110 @@
+"""
+hvPlot makes data analysis and visualization simple
+===================================================
+
+hvPlot provides an easy to use, high-level, extended 🐼 Pandas .plot like API
+that works across a wide range of data sources and plotting backends.
+
+hvPlot
+
+- supports a wide range of data sources including Pandas, Dask, XArray
+Rapids cuDF, Streamz, Intake, Geopandas, NetworkX and Ibis.
+- supports the plotting backends Bokeh (default), Matplotlib and Plotly.
+- is build on top of HoloViews and allows you to drop into HoloViews when
+more flexibility and power is needed.
+
+To learn more check out https://hvplot.holoviz.org/. To report issues or contribute check out
+https://github.com/holoviz/hvplot. To join the community check out
+https://discourse.holoviz.org/.
+
+How to use hvPlot in 3 simple steps
+-----------------------------------
+
+Work with the data source you already know and ❤️
+
+>>> import pandas as pd, numpy as np
+>>> idx = pd.date_range('1/1/2000', periods=1000)
+>>> df  = pd.DataFrame(np.random.randn(1000, 4), index=idx, columns=list('ABCD')).cumsum()
+
+Import the hvplot extension for your data source
+
+>>> import hvplot.pandas
+
+Use the `.hvplot` api as you would use the Pandas `.plot` api.
+
+>>> curves=df.hvplot()
+>>> curves
+
+In a Jupyter Notebook, this will provide you with a plot of the
+A, B, C and D time series.
+
+For more check out the user guide https://hvplot.holoviz.org/user_guide/index.html
+
+How to select the kind of plot
+------------------------------
+
+You can add the `kind` argument
+
+>>> df.hvplot(kind="scatter")
```

Comment:
What I am trying to achieve is that you can look at the docstring and learn the basics. The first thing I believe you want after seeing the basic example is to understand how to customize. And `kind` or "`.kind`" is the way to go.

I also believe its useful for users who have learned the basics but its been a while since they used hvPlot to be able to get a quick refresher.

Maybe this is too much based on my experience and too much to add?

Reply:
I've removed and simplified